app/features/build_runner/fastlane_build_runner.rb

```diff
@@ -177,6 +177,13 @@ def run(new_line_block:, completion_block:)
               original_lockfile = File.open(Bundler.default_lockfile, "wb")
               original_lockfile.write(original_lockfile_contents)
               original_lockfile.close
+              Bundler.load.clean(options[:"dry-run"])
+              Bundler.reset!
+              Bundler::Plugin.gemfile_install(Bundler.default_gemfile)
+              definition = Bundler.definition
+              definition.validate_runtime!
+              Bundler::Installer.install(Bundler.root, definition, options)
+              Bundler.require
```

Comment:
drive by review. As a non-ruby dev this has me confused. Could we add some comments 🙏

Reply:
Oh sorry, my bad

Reply:
Could you please change the lines:

```ruby
original_gemfile = File.open(Bundler.default_gemfile, "wb")
original_gemfile.write(original_gemfile_contents)
original_gemfile.close
original_lockfile = File.open(Bundler.default_lockfile, "wb")
original_lockfile.write(original_lockfile_contents)
original_lockfile.close
```

to

```ruby
File.write(Bundler.default_gemfile, original_gemfile_contents)
File.write(Bundler.default_lockfile, original_lockfile_contents)
```